drivers/led/Kconfig.is31fl319x

```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2024 Arduino SA
```

Comment:
Can you please fix the commit message ? It claims that the Kconfig file was not renamed, while it is. And also there is a lot of garbage I don't understand...

Reply:
Updated first commit message